### ISSUE TITLE
Setting arbitrary project's output path results with unworking site

### ISSUE THREAD

**Issue by vasily-kirichenko** [dotnet-websharper/suave#3]
Works ok:

![image](https://cloud.githubusercontent.com/assets/873919/11338988/3d1adfec-9208-11e5-9f33-25715a17a84f.png)

Does not work:

![image](https://cloud.githubusercontent.com/assets/873919/11339015/67f68b30-9208-11e5-9a0a-fb90ee23ff56.png)

Does not work:

![image](https://cloud.githubusercontent.com/assets/873919/11339029/7acebc50-9208-11e5-8767-85744cc925b6.png)

**Comment by Kimserey** [comment]
Somehow when you set the output path you end up with 2 different paths in the .fsproj.

```
<OutputPath>bin/Debug</OutputPath>
... further down the .fsproj ...
<OutputPath>..\..\out\Debug\MyApp</OutputPath>
```

If you delete the first one it works fine. 
After you need to specify the bin folder and root directory of your app:

```
do startWebServer defaultConfig (WebSharperAdapter.ToWebPart(Main, "root_here", "bin_here"))
```

Hope this help.

**Comment by vasily-kirichenko** [comment]
OK, thanks. 

About specifying bin and root folders by hand, should it be done automatically? I think it's very unclear and puzzling that changing project's output path results with non working site.

**Comment by vasily-kirichenko** [comment]
Ok, I've changed output path and removed duplication from the project file. Now it compiles, but does not work - the static part is loaded and then the browser is waiting for something forever.

What I've noticed, the when the output path was `bin/Debug\`, a folder named `_PublishedWebSite` (or the like) appeared there. Now it's gone - it does not exist either in `bin\Debug` nor in the new output folder (`..\..\out\Debug\Monitor.Web`).

The errors in the browser is as following:

![image](https://cloud.githubusercontent.com/assets/873919/11395135/4ede6898-937a-11e5-89c5-f566ecfab1af.png)
